### DISCUSSION TITLE
Variable input not working

### DISCUSSION THREAD

**Discussion by 0019** [grafana/grafana-infinity-datasource#118] [Q&A]
I'm using Infinity for dashboard variables, my intention is to show aliases in the variable drop down while using a less human-readable value when the variable is referenced.

My problem is that I couldn't configure the variables. After I typed in one character in the "selector" input, the input box freezes, I can't type further or delete the existing character. Moreover, it jumps back to the "Variables" page (the variable tab in the dashboard settings) when I click on "add column" (the "+" sign) or "JSON Options".

This is my screenshot:
![image](https://user-images.githubusercontent.com/12809660/128801809-a7e9553d-7489-4cfb-a4d3-821cf2026e70.png)

Could anyone help please? Did I miss configure anything? Thanks.

**Comment by yesoreyeram** [comment]
Sorry @0019 you are having trouble. This is a duplicate of #32
